application/routes.py

```diff
@@ -16,4 +17,31 @@ def index():
         return redirect(url_for('index'))
 
     to_dos = Task.query.all()
-    return render_template('index.html', to_dos=to_dos, form=form)
\ No newline at end of file
+    return render_template('index.html', to_dos=to_dos, form=form)
+
+
+@app.route('/delete/<int:task_id>')
+def delete(task_id):
+    task = Task.query.filter_by(id=task_id).first()
+    db.session.delete(task)
+    db.session.commit()
+
+    return redirect(url_for('index'))
+
+
+@app.route('/change/<int:task_id>')
+def change(task_id):
+    task = Task.query.filter_by(id=task_id).first()
```

Comment:
can you use query.get instead of query.filter?

Reply:
sure, as in the previous comment